File: DuDoong-Api/src/main/java/band/gosrock/api/cart/model/dto/request/AddCartOptionAnswerDto.java
package band.gosrock.api.cart.model.dto.request;


import io.swagger.v3.oas.annotations.media.Schema;
import javax.validation.constraints.NotBlank;
import javax.validation.constraints.NotNull;
import lombok.Getter;

@Getter
public class AddCartOptionAnswerDto {

    @Schema(description = "옵션 아이디")
    @NotNull
    private Long optionId;

    @Schema(description = "옵션 그룹에 대한 응답/ T/F면 예,아니오 ,서술형이면 서술응답", defaultValue = "예")
    @NotBlank
    private String answer;
}

File: DuDoong-Api/src/main/kotlin/band/gosrock/api/cart/model/dto/request/AddCartOptionAnswerDto.kt
package band.gosrock.api.cart.model.dto.request

import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotNull

data class AddCartOptionAnswerDto(
    @Schema(description = "옵션 아이디")
    @field:NotNull
    val optionId: Long,

    @Schema(description = "옵션 그룹에 대한 응답/ T/F면 예,아니오 ,서술형이면 서술응답", defaultValue = "예")
    @field:NotBlank
    val answer: String,
)
